irony.el

```diff
@@ -575,7 +577,8 @@ found."
        (signal (car err) (cdr err)))
       (error
        (signal 'irony-server-error
-               (if (file-executable-p exe)
+               (if (and (stringp exe)
```

Comment:
I prefer to check for `nil`, which is the sentinel value we can expect from `executable-find`:

```el
(if (and exe (file-executable-p exe))
```